### ISSUE TITLE
just a recommendation

### ISSUE THREAD

**Issue by Dcimre** [juanallo/vscode-dependency-cruiser#8]
Its not an issue, but i wwould love to be able to export at least a JPEG from the dependenciy-view its generated. :)

**Comment by vavdav** [comment]
It would be great and also zoom for big graphs :)

**Comment by juanallo** [comment]
Yes! I would love to build these features out. My main goal was to provide a UI on top that would let you do things like zoom/nav, export, filter, etc. 

I need to find time to invest into the plugin a little bit more but it has been helping me so far to do quick diagrams.

**Comment by devded** [comment]
> Its not an issue, but i wwould love to be able to export at least a JPEG from the dependenciy-view its generated. :)

yeah! it will be great if the view can be exported to JPEG

**Comment by agbozo1** [comment]
Really sleek plugin; would love an image export feature.
